docs/docs/config-file.md

```diff
@@ -0,0 +1,46 @@
+## Paths
+
+The default xemu configuration file can be found at the following locations after first run:
+
+| OS | Path |
+|----|------|
+| Linux | `~/.local/share/xemu/xemu/xemu.ini` |
+| macOS |  |
+| Windows |  |
+
+
+## Default `xemu.ini` File
+
+The default configuration file looks like this:
+
+```ini
```

Comment:
Prefer to exclude. Section is not necessary, and subject to change. Default config file will be created anyway for people who want to look at it.